Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
+
+# Create parent directory to have a place to mount pre-loaded digest files (currently the nipoppy-qpn repo)
+# TODO: Revisit this with longer-term solution to handle available digests (e.g., QPN as submodule?)
```

Comment:
agree for the future. One option could be to create a named QPN volume ahead of time, and then either manually mount it every time the container starts - or to just add a docker-compose file for this one container and define the volume in there.